talk/owt/sdk/base/peerconnectiondependencyfactory.cc

```diff
@@ -150,7 +154,10 @@ void PeerConnectionDependencyFactory::
 
 #elif defined(WEBRTC_LINUX)
   // MSDK support for Linux is not in place. Use default.
-  if (encoded_frame_) {
+  if (dual_video_encoder_) {
```

Comment:
by default, this is not enabled, so this PR is by itself a no-op. Once I get some more changes in, I will have a separate appliance PR setting this flag to true in webrtc_server's main.cc